### PR TITLE
INSTALLER: NaviGator update to 16.04

### PR DESCRIPTION
**Pull request by Grymessa**
Updated NaviGator to 16.04 and added some dependencies required by Gazebo that weren't part of the install. Catkin_Make will still fail until the main branch has been fixed to work with 16.04 as this pulls from the main branch.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -269,9 +262,9 @@ fi
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
```
Comment (kev-the-dev) [inline install.sh]: Get rid of this whole special check for INSTALL_NAV. All projects will use 16.04 or debian now

File: install.sh [hunk 2]
```diff
@@ -386,14 +379,16 @@ instlog "Installing ROS $(tr '[:lower:]' '[:upper:]' <<< ${ROS_VERSION:0:1})${RO
 sudo apt-get update -qq
 sudo apt-get install -qq ros-$ROS_VERSION-desktop-full
 
-# If ROS Indigo is being installed, break the metapackage and install an updated version of Gazebo
-if [[ "$ROS_VERSION" == "indigo" ]]; then
+# If ROS Kinetic is being installed, break the metapackage and install an updated version of Gazebo
+if [[ "$ROS_VERSION" == "kinetic" ]]; then
```
Comment (kev-the-dev) [inline install.sh]: Are you sure we need this now? I think the default kinetic gazebo is good as we used it on SubjuGator

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
See comments

File: install.sh (hunk 1 above)
Comment (Grymessa) [reply install.sh]: Kept part of it as it is used to set quite a few variables later down the line, keeping the check as install nav just works there. If preferred I can remove that and have the variables declared somewhere else or inline but it is rather neat and concise as it is now.

File: install.sh (hunk 2 above)
Comment (Grymessa) [reply install.sh]: Removed as suggested.

File: install.sh [hunk 3]
```diff
@@ -269,16 +262,6 @@ fi
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
 	REQUIRED_OS_ID="Ubuntu"
-	REQUIRED_OS_CODENAME="trusty"
-	REQUIRED_OS_RELEASE="14.04"
-	ROS_VERSION="indigo"
-elif [[ "$(lsb_release -si)" == "Debian" ]]; then
```
Comment (kev-the-dev) [inline install.sh]: Why you get rid of debian support? Should still be good

**Review by kev-the-dev — CHANGES_REQUESTED** [review]

File: install.sh (hunk 3 above)
Comment (Grymessa) [reply install.sh]: Looks like I misread that and thought it was a different version of ROS as opposed to a different version of linux. Fixing in the next commit.

File: install.sh [hunk 4]
```diff
@@ -268,22 +261,20 @@ fi
...
-	REQUIRED_OS_ID="Debian"
-	REQUIRED_OS_CODENAME="jessie"
-	REQUIRED_OS_RELEASE="8.7"
-	ROS_VERSION="kinetic"
-else
-	REQUIRED_OS_ID="Ubuntu"
-	REQUIRED_OS_CODENAME="xenial"
-	REQUIRED_OS_RELEASE="16.04"
-	ROS_VERSION="kinetic"
+	if [[ "$(lsb_release -si)" == "Debian" ]]; then
```
Comment (kev-the-dev) [inline install.sh]: Watch your indentions buddy. This file uses tabs

**Review by kev-the-dev — COMMENTED** [review]
Tabs for this file. Try to look at your entire diff on github every time you push a commit to proofread for these things

File: install.sh (hunk 4 above)
Comment (Grymessa) [reply install.sh]: Aye, sorry about that, what I get for rushing this before class. From now on I'll use git diff to see what changes I'm making and review it versus what needs to be done. That's something I should have been doing from the beginning.

File: install.sh [hunk 5]
```diff
@@ -261,20 +261,17 @@ fi
 
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
```
Comment (Grymessa) [inline install.sh]: Compared to the original installer file for reference on the format and fixed a few other things, then compared to my previous commit. The file got reverted back to original for some reason so this should fix the formatting and do everything it was supposed to do previously. If there are any other features you want me to add or programs to install let me know. I read through the file one last time and nothing sprung out as being obsolete or needing to be removed anymore. Thing builds on a Docker Image fine using your repo.

**Review by kev-the-dev — APPROVED** [review]

**Review by RustyBamboo — APPROVED** [review]

**Review by kev-the-dev — APPROVED** [review]
Let me just try building this in a fresh VM

**Comment by sentree** [comment]
The best way to test the install script is by building all of the docker images from scratch with your modified version on a test machine. Those images start out so minimal that a build on them should quickly show you any problems.

File: install.sh (hunk 4 above)
Comment (sentree) [reply install.sh]: Thanks for keeping my script pure @ironmig, #tabsforlife.

**Comment by sentree** [comment]
@Grymestone, be sure to keep the user install script in sync with the main install script. It's useful on the vehicles and workstations as it allows users to skip all of the system setup and get right to creating a workspace for the project.

**Pull request merged**